static/css/bureau.less

```diff
@@ -170,5 +171,185 @@
             .cf-icon();
         }
     }
+}
+
+// Used to help center video container
+@share-width__px: 150px;
+@gutter__em: unit(@grid_gutter-width  / @base-font-size-px, em);
+@gutter-half__em: @gutter__em / 2;
+
+.hero__bureau {
+    padding-top: 0;
+
+    .hero_wrapper {
+
+        .respond-to-min(661px, {
+            padding-top: @gutter__em * 2;
+        });
+    }
+}
+
+.hero_card__bureau {
+    .hero_preview {
+        padding-bottom: @gutter__em * 2;
+    }
+
+    .jump-link {
+        display: block;
+        margin: @gutter-half__em 0;
+    };
+}
+
+#youtube-player {
+    float: left;
+
+    .respond-to-min(1024px, {
+        padding-left: @share-width__px;
+    });
+
+    .respond-to-max(825px, {
+        .u-flexible-container_inner();
+    });
+}
+
+.youtube-player_container {
+    overflow: hidden;
+}
+
+.youtube-player_container.yt-playing {
+    background-color: #000;
+
+    .hide-on_yt-playing {
+        display: none;
+    }
+
+    .show-on_yt-playing {
+        display: block;
+    }
+
+    .youtube-player_video-container {
+        display: table;
+
+        .respond-to-max(825px, {
+            display: block;
+        })
+    }
+
+    .respond-to-max(660px, {
```

Comment:
Should perhaps be 659 and earlier 661 should be 660. Would need to be investigated.